workflow/rules/quantify.smk

```diff
@@ -109,12 +109,13 @@ rule intersect_extended_premir:
         ),
     container:
         "docker://quay.io/biocontainers/bedtools:2.30.0--h468198e_3"
+    conda:
+        os.path.join(workflow.basedir, "envs", "bedtools.yaml")
     shell:
         "(bedtools intersect \
         -wb \
         -s \
         -F 1 \
-        -sorted \
```

Comment:
Snuck that one in, eh? 😅

It's fine, but please document it in the PR description.